<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFinshStatus</class>
 <widget class="QWidget" name="QFinshStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="IconLabel">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>17</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../source.qrc">:/image/main/arraw_up.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="StatusLabel">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>13</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>上传完成</string>
   </property>
  </widget>
  <widget class="QLabel" name="DateTimeLabel">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>13</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#DateTimeLabel{color:#666666}</string>
   </property>
   <property name="text">
    <string>2017-08-22 23:59:59</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../source.qrc"/>
 </resources>
 <connections/>
</ui>
